content/compose/gettingstarted.md

```diff
@@ -214,141 +217,136 @@ image pulled from the Docker Hub registry.
 
-$ docker compose ps
+2. Cut the Redis service from your `compose.yaml` file and paste it into your new `infra.yaml` file. Make sure you add the `services` top-level attribute at the top of your file. Your `infra.yaml` file should now look like this:
 
-       Name                      Command               State           Ports         
--------------------------------------------------------------------------------------
-composetest_redis_1   docker-entrypoint.sh redis ...   Up      6379/tcp              
-composetest_web_1     flask run                        Up      0.0.0.0:8000->5000/tcp
-```
+   ```yaml
+   services:
+     redis:
+       image: "redis:alpine"
+   ```
 
-The `docker compose run` command allows you to run one-off commands for your
-services. For example, to see what environment variables are available to the
-`web` service:
+3. In your `compose.yaml` file, add the `include` top-level attribute along with the path to the `infra.yaml` file.
+
+   ```yaml
+   include:
+      - infra.yaml
+   services:
+     web:
+       build: .
+       ports:
+         - "8000:5000"
+       develop:
+         watch:
+           - action: sync
+             path: .
+             target: /code
+   ```
 
-```console
-$ docker compose run web env
-```
+4. Run `docker compose up` to build the app with the updated Compose files, and run it. You should see the `Hello world` message in your browser. 
 
-See `docker compose --help` to see other available commands.
+This is a simplified example, but it demonstrates the basic principle of `include` and how it can make it easier to modularize complex applications into sub-Compose files. For more information on `include` and working with multiple Compose files, see [Working with multiple Compose files](multiple-compose-files/_index.md).
 
-If you started Compose with `docker compose up -d`, stop
-your services once you've finished with them:
+## Step 8: Experiment with some other commands
 
-```console
-$ docker compose stop
-```
+- If you want to run your services in the background, you can pass the `-d` flag (for "detached" mode) to `docker compose up` and use `docker compose ps` to see what is currently running:
+
+   ```console
+   $ docker compose up -d
 
-You can bring everything down, removing the containers entirely, with the `down`
-command. Pass `--volumes` to also remove the data volume used by the Redis
-container:
+   Starting composetest_redis_1...
+   Starting composetest_web_1...
 
-```console
-$ docker compose down --volumes
-```
+   $ docker compose ps
+
+          Name                      Command               State           Ports         
+   -------------------------------------------------------------------------------------
+   composetest_redis_1   docker-entrypoint.sh redis ...   Up      6379/tcp              
+   composetest_web_1     flask run                        Up      0.0.0.0:8000->5000/tcp
+   ```
+
+- See `docker compose --help` to see other available commands.
```

Comment:
Nit for existing wording. It's not wrong, but the repetitive caught my eye. Maybe `use` or `run`. 
```suggestion
- Run `docker compose --help` to see other available commands.
```